### PR TITLE
Compile library using latest Android Studio Preview with androidx

### PR DESCRIPTION
**Pull request by pn**

### CODE REVIEW

**Comment by pn** [comment]
I made some changes that should make example compile on AS 3.2-beta2 (previous commits were meant only for the library) but It will still crash on runtime. To fix that h6ah4i/android-tablayouthelper needs to be migrated to androidx. I'm not planning to do it anytime soon.

File: example/build.gradle [hunk 1]
```diff
@@ -64,6 +64,6 @@ if (loadSigningConfigFromPropertiesFile(
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':library')
-    implementation 'com.android.support:design:27.0.0'
+    implementation 'com.google.android.material:material:1.0.0-beta01'
     implementation ('com.h6ah4i.android.tablayouthelper:tablayouthelper:0.9.0')
```
Comment (h6ah4i) [inline example/build.gradle]: Please tweak here like the following:
```diff
-    implementation 'com.google.android.material:material:1.0.0-beta01'
-    implementation ('com.h6ah4i.android.tablayouthelper:tablayouthelper:0.9.0')
+    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'com.h6ah4i.android.tablayouthelper:tablayouthelper:1.0.0'
```

File: library/build.gradle [hunk 2]
```diff
@@ -55,8 +55,8 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:27.0.0'
-    implementation 'com.android.support:recyclerview-v7:27.0.0'
+    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0-beta01'
```
Comment (h6ah4i) [inline library/build.gradle]: Replace these lines like this;
```diff
-    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
-    implementation 'androidx.recyclerview:recyclerview:1.0.0-beta01'
+    implementation ('androidx.recyclerview:recyclerview:1.0.0') {
+        exclude group: 'androidx.legacy', module: 'legacy-support-core-ui'
+    }
```

**Review by h6ah4i — CHANGES_REQUESTED** [review]
Thanks for your great pull request :+1: :+1: :+1: 

I noticed that some dependencies in `build.gradle` can be replaced with more better ones. If you fix them, I'll merge this PR.

**Comment by h6ah4i** [comment]
Oh, I noticed one more issue in comment blocks. Need to correct RecyclerView's package.

```diff
-androidx.appcompat.widget.RecyclerView
+androidx.recyclerview.widget.RecyclerView
```

**Comment by Suki-VT** [comment]
Is it solved ?

**Comment by pn** [comment]
@h6ah4i, thank you for review. Please check recent changes.

**Review by h6ah4i — APPROVED** [review]

**Comment by h6ah4i** [comment]
LGTM :+1: :+1: :+1:

**Pull request merged**